test/officerManager.js

```diff
@@ -103,8 +103,7 @@ describe('OfficerManager', () => {
       const form = {
         name: 'pinkUnicorn',
         email: 'test@test.com',
-        level: 2,
-        team: 'dev',
+        role: 'dev',
```

Comment:
can you use the enums here?

Reply:
yes this is a mock data, just there for placeholder